lib/components/HomePage.jsx

```diff
@@ -25,18 +26,22 @@ updateSearch(searchString) {
 	this.setState({searchString: searchString});
 }
 
+setFavorite(e) {
+	console.log('favorite')
+}
+
 renderDifficultySymbol(difficulty){
 	if (difficulty === 1) {
-		return <img className='difficulty-symbol' src = "../../images/64px-Ski_trail_rating_symbol-green_circle.svg.png"/>
+		return <img className='difficulty-symbol' src = "../public/images/green-mountain.svg"/>
```

Comment:
I changed the way difficulty is displayed. Difficulty is rated from 1-4, so it is displayed with a mountain-peak icon that is either green, yellow, orange, or red.